x/authz/keeper/grpc_query.go

```diff
@@ -33,29 +33,35 @@ func (k Keeper) Grants(c context.Context, req *authz.QueryGrantsRequest) (*authz
 	if err != nil {
 		return nil, err
 	}
-	ctx := sdk.UnwrapSDKContext(c)
-
-	store := ctx.KVStore(k.storeKey)
-	key := grantStoreKey(grantee, granter, "")
-	authStore := prefix.NewStore(store, key)
 
+	ctx := sdk.UnwrapSDKContext(c)
 	if req.MsgTypeUrl != "" {
-		authorization, expiration := k.GetCleanAuthorization(ctx, grantee, granter, req.MsgTypeUrl)
```

Comment:
so why we don't want to clean authorization if it's expired?

Reply:
Expired authorizations are not pruned from the state. see [8311#issuecomment](https://github.com/cosmos/cosmos-sdk/issues/8311#issuecomment-840997857).

Reply:
Thanks. I totally forgot about that issue.